### ISSUE TITLE
byte-array encoding/decoding is not symetrical

### ISSUE THREAD

**Issue by smessems** [dakrone/cheshire#143]
Hi, when encoding a byte-array, the internal call to writeBinary encodes it as base64.
However, the parse-string returns it as the encoded base64 string.
Maybe, this only needs to be documented, I am not sure. 

The relevant test, uses threading-last instead of threading-first macro, hence the issue is not detected ( inside test-round-trip-binary ):

`(= (->> bin-obj (get "byte-array") seq)
       (->> roundtripped (get "byte-array") seq)`

**Comment by gfredericks** [comment]
I don't think this is solvable without the user having to give information to the parser about where the byte arrays are.
